README.md

```diff
@@ -19,50 +19,56 @@
-
+A multi-backend python library for quantum optimization using QAOA on Quantum computers and Quantum computer simulators. Check out the OpenQAOA website at [https://openqaoa.entropicalabs.com/](https://openqaoa.entropicalabs.com/)
+ 
 **OpenQAOA is currently in OpenBeta.**
 
 Please, consider [joining our discord](https://discord.gg/ana76wkKBd) if you want to be part of our community and participate in the OpenQAOA's development. 
 
-Check out OpenQAOA website [https://openqaoa.entropicalabs.com/](https://openqaoa.entropicalabs.com/)
-
 ## Installation instructions
 
-You can install the latest version of OpenQAOA directly from PyPI. First, create a virtual environment with python3.8, 3.9, 3.10 and then pip install openqaoa with the following command
+OpenQAOA is divided into separately installable plugins based on the requirements of the user. The core elements of the package are placed in `openqaoa-core` which comes pre-installed with each flavour of OpenQAOA. 
+
+Currently, OpenQAOA supports the following backends and each can be installed exclusively with the exception of `openqaoa-azure` which installs `openqaoa-qiskit` as an additional requirement because Azure backends support circuit submissions via `qiskit`.
+- `openqaoa-braket` for AWS Braket
+- `openqaoa-azure` for Microsoft Azure Quantum
+- `openqaoa-pyquil` for Rigetti Pyquil
+- `openqaoa-qiskit` for IBM Qiskit
 
+The OpenQAOA metapackage allows you to install all OpenQAOA plug-ins together.
+### Install via PyPI
+You can install the latest version of OpenQAOA directly from PyPI. First, create a virtual environment with python3.8, 3.9, 3.10 and then pip install openqaoa with the following command
 ```
 pip install openqaoa
 ```
+### Install via git clone
+Alternatively, you can install OpenQAOA manually from the GitHub repository by following the instructions below. 
 
-Alternatively, you can install manually directly from the GitHub repository by
-
+**NOTE:** We recommend creating a python virtual environment for this project using a python environment manager, for instance Anaconda. Instructions can be found [here](https://conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#creating-an-environment-with-commands). Make sure to use **python 3.8** (or newer) for the environment.
 1. Clone the git repository:
-
 ```
 git clone https://github.com/entropicalabs/openqaoa.git
 ```
-
-2. Creating a python `virtual environment` for this project is recommended. (for instance, using conda). Instructions on how to create a virtual environment can be found [here](https://conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#creating-an-environment-with-commands). Make sure to use **python 3.8** (or newer) for the environment.
-
-3. After cloning the repository `cd openqaoa` and pip install the package. 
-
+2. After cloning the repository `cd openqaoa` and pip install the package as follows 
 ```
 pip install .
```

Comment:
I see. I believe this does not work anymore because now the metapackage `setup.py` does not find the plugins locally right?

Reply:
doesn't `pip install .` command installs the package from the git cloned version instead of PyPI?

Reply:
Yes to the `setup.py` pointing to the Pypi versions of the plugins.

`pip install .` runs the `setup.py` if `pip` finds one in the current repository. However, since the `setup.py` we are using in the metapackage now requires the plugins from Pypi, the install will install the version of Pypi instead of locally. (The only exception to this is if the plugins have already been installed, in that case, the metapackage won't try to install from Pypi as the plugin is already available.)

Reply:
Okay, I have removed the direct `pip install .` and instead replaced it with a new Makefile command `make local-install`. Let me know what you think

Reply:
Looks good! I like the `local-install` command